src/main/archetype/ui.frontend.react/src/components/import-components.js

```diff
@@ -13,47 +13,85 @@
  ~ See the License for the specific language governing permissions and
  ~ limitations under the License.
  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/
+import withAsyncImport from "../utils/withAsyncImport";
 
 import './Page/Page';
-import './Text/Text';
 import './Container/Container';
 import './ExperienceFragment/ExperienceFragment';
 
 import {MapTo} from '@adobe/aem-react-editable-components';
 
 import {
-    ContainerV1, ContainerV1IsEmptyFn,
-    TabsV1, TabsV1IsEmptyFn,
-    AccordionV1,AccordionV1IsEmptyFn,
-    CarouselV1,CarouselV1IsEmptyFn
-} from '@adobe/aem-core-components-react-spa';
+    ContainerV1IsEmptyFn,
+    TabsV1IsEmptyFn,
+    AccordionV1IsEmptyFn,
+    CarouselV1IsEmptyFn
+} from '@adobe/aem-core-components-react-spa/dist/isEmptyFunctions';
```

Comment:
Why are the imports done via dist?

Reply:
For the react core components library, unfortunately the code-splitted entry points are located under /dist instead of the root. I could not figure out with the time I had to have it without. While it isn't the prettiest, it does work fine however.

Reply:
https://github.com/adobe/aem-react-core-wcm-components-base/issues/2 issue created